src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -19,7 +22,148 @@ public static byte[] Protect(byte[] userData, byte[]? optionalEntropy, DataProte
             if (userData is null)
                 throw new ArgumentNullException(nameof(userData));
 
-            return ProtectOrUnprotect(userData, optionalEntropy, scope, protect: true);
+            byte[]? outputData;
+            bool result = TryProtectOrUnprotect(
+                userData,
+                optionalEntropy,
+                scope,
+                protect: true,
+                allocateArray: true,
+                bytesWritten: out _,
+                outputData: out outputData
+            );
+            Debug.Assert(result);
+            Debug.Assert(outputData != null);
+            return outputData;
+        }
+
+        /// <summary>
+        /// Encrypts the data in a specified byte span and returns a byte array that contains the encrypted data.
```

Comment:
Is there prior art on "byte span" and/or "byte array"?  At least the early API docs involving Span/ReadOnlySpan just called it "buffer".

```suggestion
        /// Encrypts the data in a specified buffer and returns an array that contains the encrypted data.
```

Is what feels "normal" to me... but I could be missing something / out of date.

Reply:
Looks like "byte array" comes from the current documentation of ProtectedData.  I'd still hedge toward "buffer" in lieu of "byte span", but am open to discussion.

Reply:
Yeah, I was trying to match the existing documentation for the bits it had. I didn't _like_ "byte span", but it was parallel with "byte array" (which I'm similarly not a huge fan of). I can easily change the ones I added to just "buffer" though.